markdown/developers/tutorials/dsl2_modules_tutorial.md

```diff
@@ -61,37 +61,58 @@ In order to create a new module, it is best to branch the code into a recognisab
 Using [nf-core/tools](https://github.com/nf-core/tools) it is very easy to create a new module. In our example, we change directory into the repository (_modules_) and we type
 
 ```bash
-nf-core modules create fgbio/fastqtobam
+nf-core modules create fgbio/demofastqtobam
 ```
 
 - The first word indicates the tool (i.e. the software or suite)
 - We separate the keys by a forward slash (`/`)
 - The second word indicates the function of the tool we are creating a module for
 
+Nf-core tools will set up an empty module for you based on a template. Follow the instructions on the command-line and a new module will be created in the fgbio folder.
+
+One parameter you have to specify during the creation is the `process resource label` based on the process label in the `base.config` files in workflows. You have to select one of the following labels:
+
+- process_single
+- process_low
+- process_medium
+- process_high
+- process_long
+- process_high_memory
+
+For now can just select the default and continue.
+
+In the next step you have to specify if you need a `meta map` with sample-specific information. Our test module will need a meta map, so select yes for the moment. If you want to know what exactly meta maps are check out the documentation [here](https://nf-co.re/docs/contributing/modules#what-is-the-meta-map).
 Magic will happen now: nf-core tools will create the following entries for the code of the module itself
 
 ```console
-software/fgbio
-└── fastqtobam
-    ├── main.nf
-    └── meta.yml
+modules
+    └── nf-core
+        └── fgbio
+            └── demofastqtobam
+                ├── main.nf
+                └── meta.yml
 ```
 
 And also the following for the testing of the module
 
 ```console
-tests/software/fgbio
-└── fastqtobam
-    ├── main.nf
-    ├── nextflow.config
-    └── test.yml
+tests
+├── modules
+│   └── fgbio
+│       └── demofastqtobam
+│           ├── main.nf
+│           ├── nextflow.config
+│           └── test.yml
+└── config
+    └── pytest_modules.yml
+
 ```
 
 Each of the files is pre-filled according to a defined nf-core template.
 
 You fill find a number of commented sections in the file, to help you modify the code while adhering to the guidelines, as you can appreciate in the following figure.
 
-![module](/assets/markdown_assets/contributing/dsl2_modules_tutorial/dsl2-mod_03_create_module.png)
+![module](/assets/markdown_assets/contributing/dsl2_modules_tutorial/dsl2-mod_03_create_module_ver2.png)
```

Comment:
could you paste the contents of this file as a markdown code block with `{nextflow}` as the language tag here instead of a screenshot?

Reply:
I have added it and removed the png file